File: js/randstring.js
var a = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
var s = '';
var s_len = 10;
for(var i = 0; i < s_len; i++){
    s += a[Math.ceil( Math.random() * (a.length-1) )];
}
console.log(s); 

File: python/randstring.py
import random
from datetime import datetime

s_len = 10
s = ''
random.seed(datetime.now())
a = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
a_len = len(a)
for i in range(s_len):
    s += random.choice(a)
print(s)
